storytime/Dockerfile

```diff
@@ -1,22 +0,0 @@
-FROM node:lts AS base
-
-RUN apt-get update && \
-    apt-get install -y && \
-    corepack enable && \
-    corepack prepare pnpm@8.7.6 --activate
-
-WORKDIR /app
-
-FROM base AS installer
-COPY . .
-RUN pnpm install --frozen-lockfile
-
-FROM base AS builder
-COPY --from=installer /app/ .
-RUN pnpm build
-
-FROM base AS app
```

Comment:
I didn't notice first time around but this didn't actually run the builder step which made this look like the build succeeded.